timesheet_automatic_reminder/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Timesheet Automatic Reminder',
+    'version': '8.0.1.0.3',

```

Comment:
@dongshukai version is always '8.0.1.0.0'
Once we merge and later new PR then version is different